app/scripts/seller-transform.js

```diff
@@ -89,6 +89,39 @@ var sellerTransform = (function(_, relatedEntityTransform, commonTransforms) {
         return buckets;
     }
 
+    function processLocationGraph(records){
+        var data = [];
+        _.each(records, function(record){
+            var point = {}
+            point.date = record.key_as_string;
+
+            point.value = {}
```

Comment:
It's helpful to use a name that's not "value" when able since "value" is a very generic term.  In this case, the object represents a mapping from city names to their count, so maybe something like "cityCounts"?